docs/1.0. Certificate Provisioning.md

```diff
@@ -221,126 +225,128 @@ If an EST Client implements the AMWA IS-04 Node API, then the EST Client SHOULD
 
 The EST Client MUST generate a new key pair every time a CSR is generated.
 
-The EST Client MUST NOT use the following hashing algorithms; MD5, SHA-0 or SHA-1 as they are no longer considered secure. EST Clients SHOULD use SHA-2 or higher, until a security vulnerability is discovered and an alternative hashing algorithm is recommended or defined in the relevant TLS specification. [NIST Policy on Hash Functions](https://csrc.nist.gov/projects/hash-functions/nist-policy-on-hash-functions).
+The EST Client MUST NOT use the following hashing algorithms, MD5, SHA-0 or SHA-1, as they are no longer considered secure. EST Clients SHOULD use SHA-2 or higher, until a security vulnerability is discovered and an alternative hashing algorithm is recommended or defined in the relevant TLS specification. Refer to the [NIST Policy on Hash Functions](https://csrc.nist.gov/projects/hash-functions/nist-policy-on-hash-functions).
 
 The CSR request SHOULD populate the "serialNumber" attribute with the device's unique serial number from [RFC 5280 - Section 4.1.2.2](https://tools.ietf.org/html/rfc5280#section-4.1.2.2)'s list of standard attributes.
 
-The CSR request SHOULD include information specific to the current authenticated TLS session within the signed CSR, this is to prevent man in the middle attacks. More information about the linking of identity and proof-of possession can be found in [RFC 7030 - Section 3.5](https://tools.ietf.org/html/rfc7030#section-3.5).
+The CSR request SHOULD include information specific to the current authenticated TLS session within the signed CSR, this is to prevent man-in-the-middle attacks. More information about the linking of identity and proof-of-possession can be found in [RFC 7030 - Section 3.5](https://tools.ietf.org/html/rfc7030#section-3.5).
 
 #### Certificate Request
 
-For each generated CSR the EST Client SHOULD make a HTTPS request containing the CSR to the `/simpleenroll` endpoint of the EST Server. The EST Client SHOULD include the manufacturer installed TLS Client Certificate if present and valid during the TLS handshake with the EST Server. If the TLS certificate is no longer valid, the [Expired Manufacturer Issued TLS Client Certificate](#Expired-Manufacturer-Issued-TLS-Client-Certificate) workflow should be followed.
+For each generated CSR, the EST Client SHOULD make an HTTPS request containing the CSR to the `/simpleenroll` endpoint of the EST Server. The EST Client SHOULD include the manufacturer-installed TLS Client Certificate, if present and valid, during the TLS handshake with the EST Server. If the TLS certificate is no longer valid, the [Expired Manufacturer-Issued TLS Client Certificate](#Expired-Manufacturer-Issued-TLS-Client-Certificate) workflow should be followed.
 
 #### Certificate Request Response
```

Comment:
```suggestion
For each generated CSR, the EST Client SHOULD make an HTTPS request containing the CSR to the `/simpleenroll` endpoint of the EST Server. The EST Client SHOULD include the manufacturer-installed TLS Client Certificate, if present and valid, during the TLS handshake with the EST Server. If the TLS certificate is no longer valid, the [Expired Manufacturer-Issued TLS Client Certificate](#expired-manufacturer-issued-tls-client-certificate) workflow should be followed.

#### Certificate Request Response
```

Reply:
I think we already did that in c01a800804603b46331cacfe22dde6765b7cf112...